torch/_dynamo/variables/builtin.py

```diff
@@ -1100,13 +1102,13 @@ def call_sum(self, tx, seq, **kwargs):
                 {},
             )
 
-    def call_reduce(self, tx, function, iterable, initializer=None):
+    def call_reduce(self, tx, function, iterable, initial=_SENTINEL):
```

Comment:
Oh, gosh this seems really hacky, but it is unfortunately, the best way to do it... Sigh...